Systems/c172p-engine.xml

```diff
@@ -121,12 +121,36 @@
             <output>/engines/active-engine/damage-level</output>
         </fcs_function>
 
-        <fcs_function>
+        <fcs_function name="volumetric factor calculation">
             <function>
-                <product>
-                    <property>/engines/active-engine/carb-volumetric-efficiency-factor</property>
-                    <property>/engines/active-engine/damage-volumetric-efficiency-factor</property>
-                </product>
+				<difference>
+					<v>1.0</v>
+					<product>
+						<property>/engines/active-engine/damage-level</property>
+						<v>0.00166</v>
+					</product>
+				</difference>
+            </function>
+            <clipto>
+                <min>0.0</min>
+                <max>1.0</max>
+            </clipto>
+            <output>/engines/active-engine/damage-volumetric-efficiency-factor</output>
+        </fcs_function>
+
+        <fcs_function name="summer for volumetric efficiency factor">
+            <function>
+                 <ifthen>
+                    <eq>
```

Comment:
Is this 3-space indentation. Can you really try to just use `<p>` instead of wrapping it in `<eq>`?